WORKSPACE

```diff
@@ -136,11 +137,13 @@ npm_translate_lock(
     patch_args = {
         "*": ["-p1"],
         "@gregmagolan/test-a": ["-p4"],
+        "meaning-of-life@1.0.0": ["-p1"],
     },
     patches = {
         "@gregmagolan/test-a": ["//examples/npm_deps:patches/test-a.patch"],
         "@gregmagolan/test-a@0.0.1": ["//examples/npm_deps:patches/test-a@0.0.1.patch"],
         "@gregmagolan/test-b": ["//examples/npm_deps:patches/test-b.patch"],
+        "meaning-of-life@1.0.0": ["//examples/npm_deps:patches/meaning-of-life@1.0.0-after_pnpm.patch"],
```

Comment:
is this entry here now redundant since the patch file is in data and listed in the lock file?

Reply:
This is to test that a patch can be applied via `patches` after a patch is applied from `pnpm.patchedDependencies`.

Reply:
ohh I see. this is a 2nd patch to show you can patch twice. 👍